litellm/__init__.py

```diff
@@ -852,7 +855,28 @@ def add_known_models():
     "petals-team/StableBeluga2",
 ]
 
-ollama_models = ["llama2"]
+# Since Ollama models are local, it is inexpensive to refresh the list at startup.
+# Also, this list can change very quickly.
+
+ollama_dir = os.environ.get("OLLAMA_MODELS", None)
+if ollama_dir is None:
+    home = Path(os.environ.get("HOME", None))
+    if home and (home / ".ollama").exists():
+        ollama_dir = home / ".ollama"
+
+if ollama_dir is None:
+    # No ollama install found.  Skip the HTTP request.
+    ollama_models = []
+else:
+    # Hit the local Ollama endpoint for a list of model names.
+    url = "http://localhost:11434/api/tags"
+    try:
+        response = requests.get(url)
+        json_dict = response.json()
```

Comment:
Note: this is fast and light, and _only_ runs if an Ollama directory is specified and present.  So it should be a no-op on systems that do not use ollama.